packages/mdc-ripple/_mixins.scss

```diff
@@ -118,23 +152,29 @@
   }
 
   &.mdc-ripple-upgraded--unbounded#{$pseudo} {
-    top: var(--mdc-ripple-top);
-    left: var(--mdc-ripple-left);
-    width: var(--mdc-ripple-fg-size);
-    height: var(--mdc-ripple-fg-size);
+    top: calc(50% - #{$radius / 2}); // fallback for browsers that don't support custom properties
```

Comment:
I don't think we need the fallback here, since this gets styled by using `mdc-ripple-upgraded--unbounded`. Since the JS checks for CSS Var support before attaching any of the `mdc-ripple-upgraded` classes, we can guarantee here that CSS variables are supported and thus you can remove this.